listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -193,14 +194,18 @@ def make_api_request(user, endpoint, **kwargs):
 def get_user_recently_played(user):
     """ Get tracks from the current user’s recently played tracks.
     """
-    return make_api_request(user, 'me/player/recently-played', limit=50)
+    latest_listened_at_ts = 0
+    latest_listened_at = user.latest_listened_at
+    if latest_listened_at:
```

Comment:
why not just `if user.latest_listened_at` here? the extra variable declaration makes this part of the code confusing, specially because there's no real difference between the names of the two variables.

Reply:
done